hw/xbox/nv2a/nv2a_pgraph.c

```diff
@@ -4080,6 +4127,116 @@ static uint8_t* convert_texture_data(const TextureShape s,
                                      unsigned int row_pitch,
                                      unsigned int slice_pitch)
 {
+    //FIXME: Handle all formats
+    if (true) {
+
+      // Get texture format information
+      ColorFormatInfo f = kelvin_color_format_map[s.color_format];
+
+      //FIXME: Extend for other formats in the main texture table
```

Comment:
This comment was probably added when `get_gl_sign_bits` wasn't a function yet.
It can be removed.